client/styles/abstracts/_variables.scss

```diff
@@ -1,4 +1,5 @@
 $base-font-size: 12;
+$sm: 576; // Small devices (landscape phones, 576px and up)
```

Comment:
why did you take out the other breakpoints? it's nice to aspire 🌈

Reply:
I will revert the changes and include only the standard breakpoints